### DISCUSSION TITLE
cross complaining protobuf-c missing libprotoc.so

### DISCUSSION THREAD

**Discussion by ghost** [protobuf-c/protobuf-c#657] [Q&A]
I am trying to cross compile protobuf-c for arm. Running ./configure works ok but when I run make I get this error:

/usr/local/lib/libprotoc.so: file not recognized: File format not recognized

I guess it is because the linker finds the the native library version. From here I went back and sucessfully cross compiled protobuf but the problem is that only the static version libprotoc.a is generated. I could not find libprotc.so. 

Your help is highly appreciated.

**Comment by lipnitsk** [comment]
Since this is a question about issues with cross-compiling `libprotoc` I suggest you raise it in the Google's [protobuf](https://github.com/google/protobuf) project
